Automattic/jetpack
# Form: Add loading state

Fixes #28472

## Proposed changes:
* Adds a blur effect to hide the form while it's loading the styles

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Create a post and add a Form block
* Check if the blur effect is being applied correctly when the form is loading the styles. 
* Reloading the page should makes it easier to observe
* Remember to test it in a published post too
